backend/src/Service/StoreOwner/StoreOwnerProfileService.php

```diff
@@ -58,6 +62,24 @@ public function getStoreOwnerProfile($userId)
         return $this->autoMapping->map('array', StoreOwnerProfileResponse::class, $item);
     }
 
+    public function getCompleteAccountStatusByStoreOwnerId($storeOwnerId): StoreOwnerCompleteAccountStatusGetResponse
```

Comment:
type hint for $storeOwnerId is missing, but ignore it for now latter can be fixed.